Sources/WasmKit/Execution/Execution.swift

```diff
@@ -7,16 +7,21 @@ import _CWasmKit
 struct Execution: ~Copyable {
     /// The reference to the ``Store`` associated with the execution.
     let store: StoreRef
+
+    /// The start of the VM stack space, used for debugging purposes.
+    let stackStart: UnsafeBufferPointer<StackSlot>
+
```

Comment:
Can you double check if we really need to add this field here? It's important to keep the `Execution` type to be <= 4 words to pass/return it directly on function calls. If we exceed that threshold, `LoadableByAddress` will demote it to pass/return indirectly via stack-allocs.